common/smoke-tests-samples/smoke-tests.yml

```diff
@@ -92,9 +92,17 @@ jobs:
           workingDir: common/smoke-test/
         displayName: List packages installed from the feed
 
-      - script: npm run smoke-test -- --devops-logging
-        workingDirectory: common/smoke-test/
-        displayName: Run smoke tests
+      - task: Maven@3
+        displayName: 'Build and Install again, JDK Version: $(JavaBuildVersion) '
+            inputs:
+                mavenPomFile: pom_generated.xml
+                goals: 'java'
```

Comment:
We'll want to use the latest dev feed versions by default: `goals: 'versions:use-latest-versions'`, similar to the [existing smoke test config](https://github.com/Azure/azure-sdk-for-java/blob/master/common/smoke-tests/smoke-test.yml#L83).